cli/cmd/generate_azure.go

```diff
@@ -0,0 +1,737 @@
+package cmd
+
+import (
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/lwgenerate/azure"
+	"github.com/pkg/errors"
+)
+
+var (
+	// Define question text here so they can be reused in testing
+	QuestionAzureEnableConfig = "Enable Azure Config Integration?"
```

Comment:
There was some contention around the use of "Config". @marcosgm requested that I rename it to `configuration` for the GCP PR. It may be the same ask here...

Reply:
Done